src/app/components/drawer/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FC } from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { Box, Text } from '@gluestack-ui/themed';
+import { DrawerContentComponentProps } from '@react-navigation/drawer';
+import { UserAvatar } from '@src/assets/svg/avatar/user-avatar';
+import {
+  HomeIcon,
+  LogoutIcon,
+  ProfileIcon,
+  SettingsIcon,
+} from '@src/assets/svg/navigation-icons/';
+
+const ROUTES = {
+  HOME: 'HomeScreen',
+  PROFILE: 'Profile',
+  SETTINGS: 'SettingsScreen',
+};
+
+interface MenuItemProps {
+  icon: JSX.Element;
+  label: string;
+  onPress: () => void;
+}
+
+const MenuItem: FC<MenuItemProps> = ({ icon, label, onPress }) => (
+  <TouchableOpacity onPress={onPress}>
+    <Box
+      flexDirection="row"
+      gap={10}
+      alignItems="center"
+      paddingVertical={17}
+      paddingHorizontal={5}>
+      {icon}
+      <Text size="md">{label}</Text>
+    </Box>
+  </TouchableOpacity>
+);
```

Comment:
In this case, we could go with the approach of creating a new directory inside this one with the name of the component. So with that approach we will have a clear component